src/main/java/seedu/mindmymoney/Storage.java

```diff
@@ -31,51 +38,42 @@ public Storage(File storageFile) {
     }
 
     /**
-     * Loads the expenditure list from the save file. If the file does not exist, or if there is
+     * Loads information from the save file. If the file does not exist, or if there is
      * an error when reading the file, return an empty list, and print a warning message.
      * @return The saved list.
      */
-    public ExpenditureList load() {
+    public User load() {
         try {
-            BufferedReader bufferedReader = new BufferedReader(new FileReader(storageFile));
-            String inputLine;
-            ArrayList<Expenditure> expenditureList = new ArrayList<Expenditure>();
-            while ((inputLine = bufferedReader.readLine()) != null) {
-                expenditureList.add(Expenditure.deserialize(inputLine));
-            }
-            bufferedReader.close();
-            return new ExpenditureList(expenditureList);
+            Scanner scanner = new Scanner(storageFile);
+            User savedUser = User.deserializeFrom(scanner);
+            scanner.close();
+            return savedUser;
         } catch (FileNotFoundException e) {
-            System.out.println("WARNING: Save file not found. MindMyMoney will create another empty expenditure list.");
```

Comment:
when we have catch errors we often throw new Mindmymoneyexception, instead of printing directly